shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -395,6 +398,12 @@ public InputConnection onCreateInputConnection(EditorInfo outAttrs) {
         return mTextInputPlugin.createInputConnection(this, outAttrs);
     }
 
+    @Override
+    public boolean checkInputConnectionProxy(View view) {
```

Comment:
Can you add a javadoc to this method?

Reply:
This is an Android method we override, there's documentation in `View#checkInputConnectionProxy`